custom_components/energidataservice/tariffs/energidataservice/chargeowners.py

```diff
@@ -24,7 +24,7 @@
     "N1": {
         "gln": "5790001089030",
         "company": "N1 A/S - 131",
-        "type": ["CD", "CD R"],
+        "type": ["CD"],
```

Comment:
FYI, the "CD R" was last used in December 2022 for discount ("rabat"). As we didn't pay tariff in that period, it contained the same value as "CD" but with the opposite (negative) sign. From August 2023 and until the end of the year, we are again not paying tariff. I didn't check if this is published in the same way (i.e. CD + CD R) or if "CD" now contains 0.

Reply:
CD R represents the discount, but as the integration is as of now, it can't handle this, which is why I removed it.

Reply:
> CD R represents the discount, but as the integration is as of now, it can't handle this, which is why I removed it.

Which actions will be taken to ensure compatibility in future versions?

Reply:
Not quite sure yet, still in the progress of thinking of a solution.
As there is no requirements as to how the operators are tagging the entries, it's just as difficult to figure out, as to how the initial module was made.